### PR TITLE
Remove field BGDIDIC-342

### PR DESCRIPTION
**Pull request by faselm**
https://mf-chsdi3.dev.bgdi.ch/feature-BGDIDIC-342/shorten/970f5c7f68

### CODE REVIEW

File: chsdi/templates/htmlpopup/fernwaerme_angebot.mako [hunk 1]
```diff
@@ -14,6 +14,5 @@
 
     <tr><td class="cell-left">${_('ch.bfe.fernwaerme-angebot.name')}</td>                         <td>${c['attributes']['name'] or '-'}</td></tr>
     <tr><td class="cell-left">${_('ch.bfe.fernwaerme-angebot.heatpotential')}</td>                <td>${heatpotential}</td></tr>
-    <tr><td class="cell-left">${_('ch.bfe.fernwaerme-angebot.heat_supplier_category')}</td>       <td>${c['attributes'][bezeichnung] or '-'}</td></tr>
```
Comment (b4l) [inline chsdi/templates/htmlpopup/fernwaerme_angebot.mako:17]: Remove line 7 as well, thanks!

**Review by b4l — CHANGES_REQUESTED** [review]

**Comment by faselm** [comment]
yep, thanks

**Comment by b4l** [comment]
@faselm so you want to remove the `bezeichnung_*` in the model as well?

**Comment by faselm** [comment]
@b4l yes, you're right it has been removed now https://github.com/geoadmin/mf-chsdi3/pull/3846

**Comment by faselm** [comment]
@b4l should be ok now

File: chsdi/templates/htmlpopup/fernwaerme_angebot.mako [hunk 2]
```diff
@@ -4,7 +4,6 @@
 
 <%
     lang = lang if lang in ('fr', 'it','en') else 'de'
```
Comment (b4l) [inline chsdi/templates/htmlpopup/fernwaerme_angebot.mako:6]: Line 6 can be removed as well ;)

**Review by b4l — APPROVED** [review]
@faselm one last dead code line to remove otherwise looking good. Regarding the `bezeichnung_*` attributes, hope this is fine for the customer like this.

**Pull request merged**